### PR TITLE
Add setting to turn on MFA

### PR DESCRIPTION
**Pull request by matmair**
This PR adds the setting for MFA to the settings page; see https://github.com/inventree/InvenTree/pull/2221

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
@matmair could you also please add:

- A note to release/0.6.0.md referencing this new feature
- A small section documenting the 2FA feature (screenshots, how to, etc)

File: docs/settings/MFA.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: InvenTree Multi Factor Authentication
+---
+
+## Multi Factor Authentication
+
+ InvenTree gives the option to use TOTP or staticly generated backup tokens as an additional factor to password or SSO authentication. This is a widely adopted security feature on enterprise web services. We highly encourage to enable it if you expose your instance to the public internet.
```
Comment (SchrodingersGat) [inline docs/settings/MFA.md:7]: "staticly" -> "statically"

File: docs/settings/MFA.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+As TOTP is an [open standard](https://datatracker.ietf.org/doc/html/rfc6238) there are a lot of different ways to hold your key and generate the time based tokens needed for authentication. That ranges from physical devices to password managers and mobile apps. We do not advertise any method but reccomend to keep password and token generator seperate from each other.
```
Comment (SchrodingersGat) [inline docs/settings/MFA.md:9]: "reccomend" -> "recommend"

File: docs/settings/MFA.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+### Configuration
+
+To make MFA mandatory for all users
+1. Enable it in the [global settings](../settings/global.md).
+
+### Security Consideration
+
+A user can lock themself out if they loose access to both the device with their TOTP app and their backup tokens. An admin can delete their tokens from the admin pages (they exsist under the 'TOTP devices' / 'static devices' models) . This should be a last resort and only done by people knowledgeable about the [admin pages](../settings/admin.md) as changes there might circumvent InvneTrees buisness and security logic.
```
Comment (SchrodingersGat) [inline docs/settings/MFA.md:18]: "loose" -> "lose"

**Comment by SchrodingersGat** [comment]
Just a couple of small spelling changes :)

**Review by SchrodingersGat — APPROVED** [review]

**Comment by SchrodingersGat** [comment]
@matmair good to go, or you want to add more?

**Comment by matmair** [comment]
> 
> 
> @matmair good to go, or you want to add more?

@SchrodingersGat still testing/fixing some fringe cases for MFA. Would only merge after MFA is merged into the main codebase.

**Comment by matmair** [comment]
@SchrodingersGat should b good to go!

**Pull request merged**